### PR TITLE
feature(ui-ux): create vault card 2.0

### PR DESCRIPTION
**Pull request by chloezxyy**
#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Loans/components/PriceOracleInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <TouchableOpacity
+        onPress={props.onPress}
+        style={tailwind("text-center flex-row items-center")}
+      >
+        <ThemedTextV2
+          dark={tailwind("text-mono-dark-v2-500")}
+          light={tailwind("text-mono-light-v2-500")}
+          style={tailwind("font-normal-v2 text-sm mr-1")}
+        >
+          {translate("", "Loan tokens get their prices from oracles.")}
```
Comment (pierregee) [inline mobile-app/app/screens/AppNavigator/screens/Loans/components/PriceOracleInfo.tsx:22]: image
should be outside dark mode background

Comment (pierregee) [reply mobile-app/app/screens/AppNavigator/screens/Loans/components/PriceOracleInfo.tsx:22]: translations missing?

File: mobile-app/app/screens/AppNavigator/screens/Loans/components/VaultBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  vaultType?: string;
+  onCardPress?: () => void;
+}): JSX.Element {
+  const { getVaultsUrl } = useDeFiScanContext();
+  return (
+    <TouchableOpacity onPress={onCardPress} activeOpacity={1}>
+      <ThemedViewV2
+        dark={tailwind("bg-mono-dark-v2-00")}
+        light={tailwind("bg-mono-light-v2-00")}
+        style={tailwind("px-5 py-4.5 rounded-lg-v2 border-0")}
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Loans/components/VaultBanner.tsx:33]:
```suggestion
        style={tailwind("p-5 rounded-lg-v2 border-0")}
```

File: mobile-app/app/screens/AppNavigator/screens/Loans/components/VaultBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                  />
+                </TouchableOpacity>
+              </>
+            )}
+            <ThemedTextV2
+              style={tailwind("font-normal-v2 text-sm text-right w-11/12")}
+            >
+              {translate("", description)}
+            </ThemedTextV2>
+            {buttonLabel !== "" && (
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Loans/components/VaultBanner.tsx:87]: should we check for `undefined` instead?

Comment (chloezxyy) [reply mobile-app/app/screens/AppNavigator/screens/Loans/components/VaultBanner.tsx:87]: can :D

**Review by pierregee — APPROVED** [review]

**Pull request merged**